### PR TITLE
Draft the blog post for Tenzir v4.19

### PR DESCRIPTION
**Pull request by dominiklohmann**

### CODE REVIEW

File: web/blog/tenzir-v4.19/index.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+      | sort timestamp
+      | chart area --position stacked
+```
+
+That's it! Our own package, all done and wrapped up.
+
+:::tip Want to dive deeper?
+We've prepared some reading material:
+- Take a look at the [`package` operator's documentation](/operators/package).
+- Read the user guide on [installing a package](/user-guides/install-a-package).
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.19/index.md:136]: @lava I think we still need to write this—we can probably take the same example that we used for this blog post, and then in addition also explain inputs and packages deployed as code.

File: web/docs/user-guides/install-a-package.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+A package definition consists of three major parts.
+
+1. Package metadata, for example the name, author and description of the
+   package.
+2. Snippets, which are example pipelines for use with the Explorer on
+   [app.tenzir.com](https://app.tenzir.com/explorer) that demonstrate how to use
+   the package.
+3. A set of inputs provided by the user to customize the package installation
+   to their environment.
+4. A set of pipelines and contexts that make up the contents of the package.
```
Comment (jachris) [inline web/docs/user-guides/install-a-package.md:17]: The guide is called "Install a Package", but starts with something that is aimed at package creators. As someone who just wants to install a package, I have to skip all the way down to "Install from the Tenzir Library".

I think this should either be separate guides, or we should switch the order, but we'd need some restructuring in that case... Alternatively, we can make this change after the next release if that's more convenient.

Comment (lava) [reply web/docs/user-guides/install-a-package.md:17]: We discussed in a call that we're leaving this as-is for now, and add some restructuring when we add a separate "Write a package" guide.

**Review by jachris — APPROVED** [review]
We reviewed this in a call.

**Pull request merged**